pkg/webui/lib/components/message/index.js

```diff
@@ -14,13 +14,21 @@
 
 import React from 'react'
 import { FormattedMessage } from 'react-intl'
+import { sanitize } from 'dompurify'
 import classnames from 'classnames'
 
 import PropTypes from '@ttn-lw/lib/prop-types'
+import replaceBackQuotes from '@ttn-lw/lib/replace-back-quotes'
 
 import style from './message.styl'
 
-const renderContent = (content, component, props) => {
+const renderHtmlContent = content => {
+  const contentWithHtml = content.map(subContent => subContent.map(replaceBackQuotes))
+
+  return sanitize(contentWithHtml)
+}
```

Comment:
Instead of doing this, you could use `import { useIntl } from 'react-intl'` to convert the message into a translated string programmatically. Then you could use a string-string-replace logic (see below) to convert that into proper react nodes, which you could then return.